src/components/common/Actions.vue

```diff
@@ -105,7 +110,8 @@ export default class Actions extends Vue {
    */
   private async onClickSave (): Promise<void> {
     this.setIsSaving(true)
-    await this.sleep(1000)
```

Comment:
Is sleep() method obsolete now? Can it be deleted now?

Reply:
Yes, done!